pkg/grizzly/server.go

```diff
@@ -181,18 +181,21 @@ func (s *Server) Start() error {
 	r.Get("/grizzly/{kind}/{name}", s.IframeHandler)
 	r.Get("/api/live/ws", livereload.LiveReloadHandlerFunc(upgrader))
 
+	_, err = s.ParseResources(s.ResourcePath)
+	if err != nil {
+		fmt.Print(err)
+	}
```

Comment:
Here, you're parsing resources for all invocations. But, with a watchScript, surely the path(s) provided are just to things to watch. We don't want to be parsing `my-dashboard.ts` for example? That's why it doesn't parse when `watchScript` is set.